dev-riscv/emulator-userspace/scripts/configure.sh

```diff
@@ -4,5 +4,9 @@ set -e
 
 rootfs="/riscv"
 
-chroot $rootfs apt-get update
-chroot $rootfs apt-get install -y build-essential git vim gdb
+run() {
+    chroot "$rootfs" /usr/bin/qemu-riscv64-static -e /usr/bin/qemu-riscv64-static "$@"
+}
+
+run /usr/bin/apt-get update
+run /usr/bin/apt-get install -y build-essential git vim gdb
```

Comment:
I got next error
```
The following packages have unmet dependencies:
 vim : Depends: libcanberra0 (>= 0.2) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
The command '/bin/sh -c /configure.sh' returned a non-zero code: 100
```
I propose to remove `vim` for a while

Reply:
Yeah, sadly this is what you get with Debian unstable :(
I'll remove it now.